.travis.yml

```diff
@@ -42,6 +43,7 @@ addons:
 
 env:
   global:
+    - DATABASE_URL=postgres://postgres:@localhost/cargo_registry_test
     - TEST_DATABASE_URL=postgres://postgres:@localhost/cargo_registry_test
```

Comment:
`DATABASE_URL` is just used by diesel for migrations by convention. The tests still run using `TEST_DATABASE_URL`